lib/boolean_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module BooleanHelper
+  def boolean_to_yesno(boo)
+    boo ? 'Yes' : 'No'
+  end
```

Comment:
This seems a bit overkill for just two variables.

```
boolean_to_yesno(boo)
```
is longer than 
```
boo ? 'Yes' : 'No'
```
and adds a layer of abstraction so you have to go look up what it does.

I understand avoiding writing logic in a view but conditionals of this simplicity seem okay.

Could you explain more why you added this help? I think I'm missing something.

Reply:
My unit test on the helper did catch a failure (I expected "Yes" but was actually returning "yes" at first). Wouldn't it be more work to write a view spec? Would I have to revert to using a fixture of markup?

I am totally open to using your suggestion, but am curious about the testing of it.

Reply:
Good morning sir. If you get a second can you share your thoughts on this ☝️.

Reply:
Ok this has been resolved. Helper and spec removed. Logic moved to email view.